docs/UserGuide.md

```diff
@@ -336,7 +354,7 @@ This command also changes the view to assignments tab (if the view is not curren
 
 ### Finding an assignment by keywords
 
-```put command first```
+```find-a [keyword]```
```

Comment:
capitalise KEYWORD to be consistent and is keyword optional? if it is compulsory you shouldn't put the square brackets